src/EA.Weee.Requests/AatfEvidence/GetAatfNotesRequest.cs

```diff
@@ -21,11 +21,15 @@ public class GetAatfNotesRequest : EvidenceNoteFilterBaseRequest
 
         public DateTime? EndDateSubmitted { get; set; }
 
+        public int? SelectedComplianceYear { get; set; }
+
+        public DateTime? CurrentDate { get; set; }
+
         public GetAatfNotesRequest(Guid organisationId, 
             Guid aatfId, 
             List<NoteStatus> allowedStatuses,
             string searchRef, Guid? recipient, WasteType? wasteType, NoteStatus? noteStatusFilter,
-            DateTime? startDateSubmitted, DateTime? endDateSubmitted)
+            DateTime? startDateSubmitted, DateTime? endDateSubmitted, int? selectedComplianceYear, DateTime? currentDate)
```

Comment:
selected complianceYear here shouldnt be nullable. For these requests now it is always required.